addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -391,27 +380,53 @@ function registerBackendAndFrontendTour(name, options, steps) {
     }, steps);
 }
 
-/**
- * Selects an element inside a we-select, if the we-select is from a m2o widget, searches for it.
- *
- * @param widgetName {string} The widget's data-name
- * @param elementName {string} the element to search
- * @param searchNeeded {Boolean} if the widget is a m2o widget and a search is needed
- */
-function selectElementInWeSelectWidget(widgetName, elementName, searchNeeded = false) {
-    const steps = [clickOnElement(`${widgetName} toggler`, `we-select[data-name=${widgetName}] we-toggler`)];
-
+  /**
+   * Selects an element inside a we-select, if the we-select is from a m2o widget, searches for it.
+   *
+   * @param widgetName {string} The widget's data-name
+   * @param elementName {string} the element to search
+   * @param searchNeeded {Boolean} if the widget is a m2o widget and a search is needed
+   */
+  function selectElementInWeSelectWidget(
+    widgetName,
+    elementName,
+    searchNeeded = false
+  ) {
+    const we_select = `we-select[data-name=${widgetName}]`;
+    const steps = [
+      {
+        content: `Clicking on the ${widgetName} toggler in vue to select ${elementName}`,
+        trigger: `${we_select} we-toggler`,
+        run: "click",
+      },
+    ];
     if (searchNeeded) {
-        steps.push({
-            content: `Inputing ${elementName} in m2o widget search`,
-            trigger: `we-select[data-name=${widgetName}] div.o_we_m2o_search input`,
-            run: `text ${elementName}`
-        });
+      steps.push({
+        content: `Inputing ${elementName} in m2o widget search`,
+        trigger: `${we_select} div.o_we_m2o_search input`,
+        run: `text ${elementName}`,
+      });
     }
-    steps.push(clickOnElement(`${elementName} in the ${widgetName} widget`,
-        `we-select[data-name=${widgetName}] we-button:contains(${elementName})`));
+    steps.push({
+      content: `Clicking on the ${elementName} in the ${widgetName} widget`,
```

Comment:
Whole indentation is broken, I'll check once that is fixed, thank you :pray:

Reply:
the tour fails once every 650 times... it takes time to check that it's ok